index.bs

```diff
@@ -1329,8 +1329,12 @@ Removing AB or TAG members</h5>
 	Individual participants of the [=Advisory Board=] or the [=Technical Architecture Group=]
 	can be <dfn export lt="remove|removal">removed</dfn> from those groups
 	if they are found by their peers
-	to be grossly neglecting their duties,
-	or to be acting in a way that seriously hampers the group's ability to function.
+	to be grossly neglecting their duties
+	or acting in a way that seriously hampers the group's ability to function;
+	or if they are violating policies or other standards of behavior
+	(such as applicable laws or the <a href="https://www.w3.org/policies/code-of-conduct/#Reporting">Code of Conduct</a>)
+	in a way that significantly impacts the safety or integrity
+	of W3C or its community.
```

Comment:
This is interesting. It sidesteps the issue of how a potential CoC violation should be handled, which reflects the concerns described previously. However, to do that, it adds a new condition "a way that significantly impacts the safety or integrity of W3C or its community", which is likely to be a subjective metric, and is seemingly to be assessed by the individual's peers in their group.

I'm not sure how I feel about that, and welcome the views of others. I'm certainly not instantly comfortable with it.